### PR TITLE
Fix bucket trigger position

### PR DESCRIPTION
**Pull request by romeovs**
Fixes a bug where the bucket popover trigger is positioned incorrectly in the query column picker.

Screenshot 2024-05-23 at 13 09 44

### How to verify

Describe the steps to verify that the changes are working as expected.

1. New question -> Sample Dataset -> People
2. Add an aggregation
3. Hover a date/time column
4. Verify the bucket picker is in the correct location

### Demo

Screenshot 2024-05-23 at 13 09 23

### CODE REVIEW

File: frontend/src/metabase/common/components/QueryColumnPicker/QueryColumnPicker.styled.tsx [hunk 1]
```diff
@@ -14,12 +14,15 @@ export const ColumnNameContainer = styled.div`
   display: flex;
   align-items: center;
   flex-shrink: 0;
+  width: 100%;
```
Comment (kamilmielnik) [inline frontend/src/metabase/common/components/QueryColumnPicker/QueryColumnPicker.styled.tsx:17]: The inner border/separator is not full-height: [this PR](https://github.com/metabase/metabase/assets/6830683/788fd45a-c957-4b58-b24a-7e6ddaf630ab) vs [0.49.11](https://github.com/metabase/metabase/assets/6830683/7e361c36-b617-4d45-b38a-c5a728af7fcf)

**Review by ranquild — COMMENTED** [review]
@romeovs I think that custom styles were required when we positioned the label differently. We could revert everything to what it was before in 48. Could you check that?

**Comment by romeovs** [comment]
@ranquild a full (git) revert is not possible since we still need some of the new functionality from before. I've reverted the relevant styles and markup to match what it was before.

**Review by kamilmielnik — APPROVED** [review]
LGTM :+1:

**Pull request merged**